sample_pages/esp/eids-esp.1.0.js

```diff
@@ -0,0 +1,26 @@
+function fetchAsyncSignals(mode) {
+        switch (mode) {
+        case 1:
+          var eids=""
+          if(pbjs && pbjs.getUserIdsAsEids && typeof pbjs.getUserIdsAsEids === "function"){
+            eids=pbjs.getUserIdsAsEids(); // Get Identities from Prebid API in oRTB eids structure
+          }
+          break;
+        case 2:
+          var eids=""
+          if(owpbjs && owpbjs.getUserIdsAsEids && typeof owpbjs.getUserIdsAsEids === "function"){
+            eids=owpbjs.getUserIdsAsEids(); //Get Identities from Identity Hub  API in oRTB eids structure
+          }
+          break;
+        default:
+          eids="Hello eids"; // Demo data
+         
+      }
+      signals= encryptSignals(JSON.stringify({"eids":eids}))
+      promise = Promise.resolve(signals);       
+      return promise;
+};
+
+function encryptSignals(signals){
+    return btoa(signals); // Test encryption. To be replaced with better algo
```

Comment:
![Codacy](https://codacy.pubmatic.com:443/assets/images/favicon.png) Issue found: [Expected indentation of 1 tab but found 4 spaces.](https://codacy.pubmatic.com:443/manual/dattaprasad.mundada/OpenWrap/pullRequest?prid=15858)